src/app/pages/arbol-cuentas-contables/arbol-cuentas-contables.component.html

```diff
@@ -1,69 +1,39 @@
-            {{'GLOBAL.ver' | translate }}
-          </button>
+        <div class="tab-title">
+          {{tab.title}}
         </div>
-        <div class="col">
-          <button [disabled]="!selectedNodeData || selectedNodeData?.Hijos" type="submit" class="btn btn-success btn-block"
-            (click)="getTreeInfo(); makeFormEditable(false, true); showViewTab('group');">
-            {{'GLOBAL.editar' | translate }}
-          </button>
+        <div>
+          <nb-icon [icon]="'times'" aria-hidden="true" pack="fas" status="secondary" (click)="removeTab(index)">
+          </nb-icon>
         </div>
+      </ng-template>
+      <div style="padding: 20px;">
+        <ngx-cuenta-contable [cuenta]="tab.type == 'crear' ? null : selectedCoumt"></ngx-cuenta-contable>
       </div>
-      <br>
-      <label class="search-label" for="search">{{
-      "GLOBAL.buscar" | translate
-    }}</label>
-
-      <input nbInput [nbFilterInput]="dataSource" id="search" class="search-input" [(ngModel)]="searchValue" />
-      <table [nbTreeGrid]="dataSource" [nbSort]="dataSource" (sort)="updateSort($event)">
-        <tr nbTreeGridHeaderRow *nbTreeGridHeaderRowDef="allColumns"></tr>
-        <tr #nebularTree class="table-row" [class.highlight]="validHighlight(row, nebularTree)"
-          (click)="onSelect(row, nebularTree);" nbTreeGridRow *nbTreeGridRowDef="let row; columns: allColumns"
-          [attr.data-picker]="row.data.Codigo"></tr>
-        <ng-container [nbTreeGridColumnDef]="customColumn">
-          <th nbTreeGridHeaderCell [nbSortHeader]="getSortDirection(customColumn)" *nbTreeGridHeaderCellDef>
-            {{ customColumn }}
-          </th>
-          <td class="aligned-code" nbTreeGridCell *nbTreeGridCellDef="let row">
-            <ngx-nb-fs-icon [kind]="row.hasChildren() ? 'dir' : 'doc'" [expanded]="row.expanded"></ngx-nb-fs-icon>
-             {{ row.data[customColumn] }}
-          </td>
-        </ng-container>
-        <ng-container *ngFor="let column of defaultColumns; let index = index" [nbTreeGridColumnDef]="column"
-          [showOn]="getShowOn(index)">
-          <th nbTreeGridHeaderCell [nbSortHeader]="getSortDirection(column)" *nbTreeGridHeaderCellDef>
-            {{ column }}
-          </th>
-          <td nbTreeGridCell *nbTreeGridCellDef="let row">
-            {{ column === "Activo" ? (row.data[column] ? "Si" : "No" ): (row.data[column] || "-") }}
-          </td>
-        </ng-container>
-      </table>
-    </div>
-    <button *ngIf="viewTab" (click)="cleanInterface(); showTreeTab();" class="btn btn-info btn-sm"> <i class="nb-arrow-dropleft"></i>
-      Regresar</button>
-    <ngx-dinamicform *ngIf="viewTab" [normalform]="formData" [modeloData]="nodeData" (result)="validarForm($event)"
-      (resultChanges)="validarCampo($event)" [clean]="cleanForm">
-    </ngx-dinamicform>
-  </nb-card-body>
-</nb-card>
\ No newline at end of file
+    </mat-tab>
+  </mat-tab-group>
+</div>
```

Comment:
(En general) Dejar vacía, sin espacios ni nada, la última línea de cualquier archivo de texto plano
```suggestion
</div>

```